lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -1553,10 +1553,15 @@ getJormungandrBlock0H = do
 prepExternalTxViaJcli :: Port "node" -> Text -> Natural -> IO Text
 prepExternalTxViaJcli port addrStr amt = do
     withTempDir $ \d -> do
+        let (<//>) dir name = T.unpack $ (T.pack dir) </> (name :: Text)
```

Comment:
This was slightly hellish

- We need to write the key to a file.
- To replace `<> "/"` with `</>` I had lots of problems with `Text`-`String` conversions, so I defined a helper operator (Actually, now when I look at it, I don't understand why I had to… Everything should already be using `String`🤔 )

Reply:
AH!

It is not [System.Directory.</>](http://hackage.haskell.org/package/filepath-1.4.2.1/docs/System-FilePath-Posix.html#v:-60--47--62-), but:
```
(</>) :: ToHttpApiData a => Text -> a -> Text
base </> next = mconcat [base, "/", toQueryParam next]
```